docs/service-account-creation.md

```diff
@@ -1,4 +1,13 @@
-# Introduction
+## Creating A Service Account
 
 In order to deploy images or infrastructure using terraform to your Google Cloud Platform project, it is important that you create a service account and download a keyfile.
 This guide hopes to guide you through that process
+1. Navigate to the **Service Accounts** page on GCP console
```

Comment:
Navigating to the **`Service Accounts`** page for a first time user might seem difficult. It would be more helpful to include a description like **`navigation menu > IAM & admin > service account`**

Reply:
Actually I believe you should (as much as possible) include images (screenshots)
Not just for this but for other docs